apps/wallet/src/ui/app/shared/ButtonUI.tsx

```diff
@@ -26,7 +26,7 @@ const styles = cva(
 					'disabled:bg-hero-darkest disabled:text-white disabled:opacity-40',
 				],
 				secondary: [
-					'bg-gray-40 text-steel-darker border-none',
+					'bg-hero-darkest/10 text-steel-darker border-none',
```

Comment:
Confirmed with Abhi that we want to update the secondary button color to this

Edit: Just remembered that we also need to update the hover/focus/disabled states - will do that now